data/starts.txt

```diff
@@ -0,0 +1,208 @@
+	`	Those anxious musings are interrupted when the bank employee finally lays it out to you, with a hesitant smile. "You are accepting a loan in the amount of 1,431,000 credits, due over the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
+	name
+	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is selling off his fleet at a reduced price, and one of the ships he's offering is a Grasshopper... maybe you could get a deal on it.`
+	choice
+		`	(Go to the shipyard.)`
+			decline
+		`	(Go see what the captain has to offer first.)`
+	`	You head to the spaceport and easily find the captain, a Hai, selling one of his Centipedes to another Hai. Thankfully it looks like he has not sold the Grasshopper yet.`
+	`	When he's finished selling the Centipede the captain turns and greets you in Hai. You are embarrassed to have to explain to them that you never learned their language. Most Hai speak your language however, and, as usual, they are able to switch to speaking yours without much difficulty.`
+	`	"Hello, good fellow! I assume you come for the Grasshopper." Your interest for this ship must be apparent.`
+	choice
+		`	"Indeed."`
+		`	"How did you know?"`
+		`	"Sorry, not interested."`
+			decline
+	`	"I can see the will to explore in your eyes, but it is not only that: I know that some humans want to head back to their space, now that it is safer to do so. I would be delighted to have you flying the 'Pristine Jewel'!" he says, taking you to the ship. It is clearly not brand new, but is still in a surprisingly good condition.`
+	scene "thumbnail/hai grasshopper"
+	`	"Because I like you I'll give you a friendly price. What do you say about... a bit over a million credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
+	choice
+		`	"Sold!"`
+		`	"Sorry, not interested."`
+			decline
+	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. In your excitement to buy the ship all you can hear from the Hai captain's speech is the repeated reference to the Grasshopper's 'exceptional' reinforced armor.`
+	`	Before you leave, he gives you a final tip: "If you want to, you can always sell this ship and buy an Aphid instead. The shipyard authorities will give you a discount for having sold my ship."`
```

Comment:
Talking about how the player doesn't know Hai or talking about how the Grasshopper is customised are some of the most boring things you could do in this situation.

The player not knowing Hai is such a generic detail that you could write almost any interaction with a Hai and put that line in. About the only place where it'd be more than just a band-aid explanation over a plot hole is maybe in the bank scene, where you could get some tension about not knowing the Hai language when dealing with very important legal papers.

The Grasshopper modifications are more relevant to the situation, but the line's only purpose is to go "HEY! THE STATS ON THIS VIDEO GAME SHIP ARE DIFFERENT FROM THE STATS OF OTHER SIMILAR SHIPS IN THIS VIDEO GAME", which isn't great for immersion. Obviously, you do need to convey this info to the player, but you can just sneak it in as an aside instead of making it the ship's main description.

What hasn't been explored at all in this segment is why he's even retiring in the place. Is it because of old age? Has the lifestyle become boring to him? Was he traumatised by a freak Unfettered attack? Is the rationale behind his retirement the same as a human might have?

And also, what are people doing in response to the captain selling of his fleet? (Besides the obvious answer of "buying his ships".) Do they see it as a noble way to give back to the people? Or do they think that it's a gaudy stunt, and that he should just be selling it straight to a shipyard? Are they picking up his ships left, right and centre, or is he having to offer ridiculous discounts just to get rid of his ships? How does Hai society interact with the situation in a way that human society doesn't?

Answering these questions is what will make this start a proper introduction to the universe from the perspective of a human in Hai space, and not just a linear sequence of events and thoughts that are inflicted upon the player.

Reply:
I do need to say the player doesn't speak hai, or give them a translator which potentially wouldn't mix that well with campaigns and all, as only the uhai one takes that into account that I know of